Assets/ConsentManagementProvider/Scripts/json/JsonUnwrapper.cs

```diff
@@ -260,30 +264,36 @@ private static GdprConsent UnwrapGdprConsent(GdprConsentWrapper wrapped)
 
         private static ConsentStatus UnwrapConsentStatus(ConsentStatusWrapper wrappedconsentStatus)
         {
-            GranularStatus granularStatus = null;
```

Comment:
@Nevazhnovu have a look at the USNAT consent status object [`SPUSNatConsent.Statuses`](https://github.com/SourcePointUSA/ios-cmp-app/blob/8ca12cce373dede943cb2856e609d11b8c4c6e95/ConsentViewController/Classes/Consents/SPUSNatConsent.swift#L180). It changed shape and I believe you should not re-use the `ConsentStatus` class we previously had.

Reply:
Ah never mind, I see you already created a different `ConsentStatus` class for `UsNat` 👍